src/tactic/interactive_expr.lean

```diff
@@ -390,8 +391,16 @@ tc.stateless $ λ ft, do
   let lcs := group_local_collection lcs,
   lchs ← lcs.mmap (λ lc, do
     lh ← local_c lc,
+    cls ← is_class lc.type,
     let ns : list (html γ) := lc.locals.map $ λ n,
-      h "span" [cn "goal-hyp b pr2", key n.local_uniq_name] [html.of_name n.local_pp_name],
+      let var_style : string := if ¬ cls
+        then "goal-hyp"
+        else match is with
+        | some is' := if n ∈ is'
+                        then "goal-hyp-inst"
+                        else "goal-hyp-noninst"
+        | none := "goal-hyp-unfrozen"
```

Comment:
In the interest of backwards-compatibility with older extension versions, perhaps
```suggestion
        then "goal-hyp"
        else match is with
        | some is' := if n ∈ is'
                        then "goal-hyp goal-hyp-inst"
                        else "goal-hyp goal-hyp-noninst"
        | none := "goal-hyp goal-hyp-unfrozen"
```

Reply:
Would it make sense to do
```suggestion
        then "goal-hyp"
        else match is with
        | some is' := if n ∈ is'
                        then "goal-hyp inst"
                        else "goal-hyp noninst"
        | none := "goal-hyp unfrozen"
```
instead?

Reply:
No, because `inst` is a pretty vague CSS class name, as is `unfrozen`. If in doubt, qualify your names.

Reply:
Won't the fully qualified name be `goal-hyp inst`? Disclaimer: I don't understand CSS.

Reply:
`class` (which `cn` sets) is an unordered `set` of unrelated classnames. You can of course apply styles to things that match `goal-hyp` and `inst`, but the danger is that someone else might already be applying a style you don't expect to `inst` by itself.